packages/components-react/src/Avatar.tsx

```diff
@@ -0,0 +1,26 @@
+import { IconBuilding, IconUser } from '@tabler/icons-react';
+import { HTMLAttributes, PropsWithChildren } from 'react';
+import * as React from 'react';
+
+export interface AvatarProps extends HTMLAttributes<HTMLButtonElement> {
+  imageSrc?: string;
+  altText?: string;
+  privateProfile?: boolean;
+}
+export const Avatar = ({ children, imageSrc, altText, privateProfile, ...props }: PropsWithChildren<AvatarProps>) => {
+  return (
+    <button className={'kernteam-avatar'} {...props}>
```

Comment:
Kun je eens opzoeken of hier een ESLint regel voor is?
`className={'kernteam-avatar'}` ➝ `className='kernteam-avatar'`